olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -820,7 +820,7 @@ spec:
                 serviceAccountName: knative-operator
                 containers:
                   - name: knative-operator
-                    image: registry.ci.openshift.org/knative/main:serverless-openshift-knative-operator
+                    image: registry.ci.openshift.org/knative/auto/bump-so-version-release-1.33:serverless-openshift-knative-operator
```

Comment:
That does not seem right? @pierDipi , right?

Reply:
Regex are too wide for the workflow

Reply:
Hm, but bumping for the non-main images would be correct for the release branch. or is that another Workflow?

Reply:
Yes, it's correct, and that's what being triggered I guess we need the regex to match the exact starting of the branch name `^release-`